packages/camera/camera_web/example/integration_test/types/camera_metadata_test.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:camera_web/src/types/types.dart';
+import 'package:flutter_test/flutter_test.dart';
```

Comment:
This test can be moved to camera_web/test and run as a VM test, no need to run this as an integration test.

Reply:
Good point!